scout/server/blueprints/cases/views.py

```diff
@@ -725,7 +724,11 @@ def custom_report(institute_id, case_name, report_type):
         try:
             with open(os.path.abspath(report_path), "r") as html_file:
                 source_code = html_file.read()
-                if CUSTOM_CASE_REPORTS[report_type]["format"] == "YAML":
+
+                if (
+                    CUSTOM_CASE_REPORTS[CUSTOM_CASE_REPORT_KEY_NAMES[report_type]]["format"]
```

Comment:
This works, but the syntax is quite complex with a very nested dictionary. I don't like much my solution but I find it simpler to follow and to maintain?

Reply:
I prefer this one, but I'm ok with your solution just checking file ending. But I would prefer it if we at least also fix the endpoint test then to use the keys the endpoint actually receives...

Reply:
I can add some parametrized tests tomorrow!

Reply:
It has a parametrised test already; just needs the proper key - but you can e.g. reuse the reverse dict from here, or make a list directly from the original dict member dicts key_name members!

Reply:
But Ok, I'll check the code tomorrow!